src/openapi/streaming/streaming.ts

```diff
@@ -639,6 +640,9 @@ class Streaming extends MicroEmitter<EmittedEvents> {
                 this.handleControlMessageDisconnect();
                 break;
 
+            case OPENAPI_CONTROL_MESSAGE_CONNECTION_HEARTBEAT:
+                // control mesage to keep streaming connection alive
+                break;

```

Comment:
nitpick - new line after the break to be consistent

Reply:
added new line